packages/create-astro/test/install-step.test.js

```diff
@@ -26,6 +26,7 @@ describe('[create-astro] install', function () {
 			const seen = new Set();
 			const installPrompt = PROMPT_MESSAGES.install(FAKE_PACKAGE_MANAGER);
 			stdout.on('data', (chunk) => {
+				console.log(chunk.toString());
```

Comment:
Looks like we need to disable ESLint for this line:

```suggestion
				// eslint-disable-next-line no-console
				console.log(chunk.toString());
```

Reply:
Or was it just for debugging and should be removed?

```suggestion
```